### PR TITLE
3 new scripts

### PR DESCRIPTION
**Pull request by mlbright**
* rotate logs to save disk space on root partition
* restart management console after replica promotion

### CODE REVIEW

File: admin/delete-ghost-repo.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+done
+
+REPO=$1
+if [ -z "$REPO" ]; then
+    usage "source org/repo is a required parameter!"
+fi
+
+execute << EOF
+    echo "Repository.find_by_name_with_owner('$REPO').remove(User.find_by_login('ghost'))" | ghe-console -y
+EOF
```
Comment (larsxschneider) [inline admin/delete-ghost-repo.sh:34]: That doesn't remove the ghost repo. It removes a repo with the special user ghost.
I think this was kind of a special case... I don't think we will use this again (at least I hope so)

File: admin/restart-management-console.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        (-*) usage "$1: unknown option";;
+        (*) break;;
+    esac
+done
+
+execute << EOF
+    sudo service enterprise-manage stop
+    sleep 10
+    sudo service enterprise-manage start
+EOF
```
Comment (larsxschneider) [inline admin/restart-management-console.sh:31]: When would we restart the console?

File: admin/rotate-logs.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        (--) shift; break;;
+        (-*) usage "$1: unknown option";;
+        (*) break;;
+    esac
+done
+
+echo "Rotating the GitHub logs ..."
+execute << EOF
+    sudo logrotate -v -f /etc/logrotate.conf
+EOF
```
Comment (larsxschneider) [inline admin/rotate-logs.sh:30]: I like this script! Can you make a PR with just this?

File: admin/restart-management-console.sh (hunk 2 above)
Comment (mlbright) [reply admin/restart-management-console.sh:31]: Possibly after promoting a replica to a primary.

File: admin/delete-ghost-repo.sh (hunk 1 above)
Comment (mlbright) [reply admin/delete-ghost-repo.sh:34]: OK will remove this script. (And seperate the other two.)

**Comment by mlbright** [comment]
Created separate PRs for the scripts still in contention.

File: admin/restart-management-console.sh (hunk 2 above)
Comment (larsxschneider) [reply admin/restart-management-console.sh:31]: What if the add the replica to https://github.com/larsxschneider/scotty/blob/master/ghe.config.template and then add a command `admin/switch-to-replica.sh` or something?

**Pull request closed without merging**